apps/crn-server/test/data-providers/contentful/teams.data-provider.test.ts

```diff
@@ -0,0 +1,583 @@
+import { GraphQLError } from 'graphql';
+import {
+  getContentfulGraphqlClientMockServer,
+  Environment,
+} from '@asap-hub/contentful';
+
+import {
+  getContenfulGraphqlTeam,
+  getContentfulTeamsGraphqlResponse,
+  getTeamCreateDataObject,
+  getTeamDataObject,
+} from '../../fixtures/teams.fixtures';
+import { getContentfulGraphqlClientMock } from '../../mocks/contentful-graphql-client.mock';
+import { getContentfulEnvironmentMock } from '../../mocks/contentful-rest-client.mock';
+import {
+  TeamContentfulDataProvider,
+  teamUnreadyResponse,
+} from '../../../src/data-providers/contentful/teams.data-provider';
+import { getEntry } from '../../fixtures/contentful.fixtures';
+
+describe('Teams data provider', () => {
+  const contentfulGraphqlClientMock = getContentfulGraphqlClientMock();
+  const environment = getContentfulEnvironmentMock();
```

Comment:
I'd do
```suggestion
  const environmentMock = getContentfulEnvironmentMock();
```
only because later you will be adding expectations on its methods. It's therefore important to make it obvious that this is a mock so that you don't need to use spies (which you actually did use - see my comment down below)